src/cloud-api-adaptor/ibmcloud/image/push.sh

```diff
@@ -95,7 +95,13 @@ fi
 image_ref="cos://$location/$cos_bucket/$object_key"
 
 arch=$(uname -m)
-[ "${SE_BOOT:-false}" = "true" ] && os_name="hyper-protect-1-0-s390x" || os_name="ubuntu-20-04-${arch/x86_64/amd64}"
+if [[ "${SE_BOOT:-false}" = "true" ]]; then
+    os_name="hyper-protect-1-0-s390x"
+elif [[ ${arch} == "s390x" ]]; then
+    os_name="ubuntu-22-04-s390x"
+else
+    os_name="ubuntu-22-04-${arch/x86_64/amd64}"
```

Comment:
I'm a bit confused about the 22-04 vs 24-04 part. Shouldn't this one be 24-04?
```suggestion
    os_name="ubuntu-24-04-${arch/x86_64/amd64}"
```

Reply:
Yes - it should!

Reply:
All the doc md files referring to 22-04 is a little confusing, but I guess that will get straightened out with s390x has 24-04. Otherwise, the code LGTM.

Reply:
Yeah - it's not ideal that IBM Cloud still doesn't have 24-04 images for s390x, but maybe 20.04 going EoL (without extended support contracts) might prompt that 🤞